meter/validation.cc

```diff
@@ -37,6 +47,8 @@ static bool is_user_key_invalid(const std::string& k) noexcept {
   return false;
 }
 
+bool empty_or_null(util::StrRef r) { return r.empty() || r.length() == 0; }
```

Comment:
I found this a bit confusing, `empty()` on the reference means the string is null. My first assumption is it would mean an empty string `""`.